To: vim_dev@googlegroups.com
Subject: Patch 7.3.247
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.247
Problem:    Running tests changes the users viminfo file.  Test for patch 
            7.3.246 missing.
Solution:   Add "nviminfo" to the 'viminfo' option.  Include the test.
Files:      src/testdir/test78.in, src/testdir/test81.in


*** ../vim-7.3.246/src/testdir/test78.in	2011-06-13 01:07:22.000000000 +0200
--- src/testdir/test78.in	2011-07-15 13:26:22.000000000 +0200
***************
*** 6,12 ****
  
  STARTTEST
  :so small.vim
! :set nocp fileformat=unix undolevels=-1
  :e! Xtest
  ggdG
  :let text = "\tabcdefghijklmnoparstuvwxyz0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnoparstuvwxyz0123456789"
--- 6,12 ----
  
  STARTTEST
  :so small.vim
! :set nocp fileformat=unix undolevels=-1 viminfo+=nviminfo
  :e! Xtest
  ggdG
  :let text = "\tabcdefghijklmnoparstuvwxyz0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnoparstuvwxyz0123456789"
*** ../vim-7.3.246/src/testdir/test81.in	2011-06-26 05:36:07.000000000 +0200
--- src/testdir/test81.in	2011-07-15 13:25:00.000000000 +0200
***************
*** 1,10 ****
  Test for t movement command and 'cpo-;' setting
  
  STARTTEST
! :set nocompatible
  :set cpo-=;
  /firstline/
  j0tt;D
  $Ty;D:set cpo+=;
  j0tt;;D
  $Ty;;D:?firstline?+1,$w! test.out
--- 1,12 ----
  Test for t movement command and 'cpo-;' setting
  
  STARTTEST
! :set nocompatible viminfo+=nviminfo
  :set cpo-=;
  /firstline/
  j0tt;D
+ 0fz;D
+ $Fy;D
  $Ty;D:set cpo+=;
  j0tt;;D
  $Ty;;D:?firstline?+1,$w! test.out
***************
*** 13,18 ****
--- 15,22 ----
  
  firstline
  aaa two three four
+     zzz
+ yyy   
  bbb yee yoo four
  ccc two three four
  ddd yee yoo four
*** ../vim-7.3.246/src/version.c	2011-07-15 13:21:24.000000000 +0200
--- src/version.c	2011-07-15 13:31:14.000000000 +0200
***************
*** 711,712 ****
--- 711,714 ----
  {   /* Add new patch number below this line */
+ /**/
+     247,
  /**/

-- 
ZOOT:  I'm afraid our life must seem very dull and quiet compared to yours.
       We are but eightscore young blondes, all between sixteen and
       nineteen-and-a-half, cut off in this castle, with no one to protect us.
       Oooh.  It is a lonely life ... bathing ...  dressing ... undressing ...
       making exciting underwear....
                 "Monty Python and the Holy Grail" PYTHON (MONTY) PICTURES LTD

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
